src/psyclone/psyad/transformations/__init__.py

```diff
@@ -0,0 +1,42 @@
+# BSD 3-Clause License
+#
+# Copyright (c) 2021, Science and Technology Facilities Council.
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of source code must retain the above copyright notice, this
+#   list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# * Neither the name of the copyright holder nor the names of its
+#   contributors may be used to endorse or promote products derived from
+#   this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+# "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+# LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+# FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+# COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+# INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+# BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+# LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+# LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+# ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+# -----------------------------------------------------------------------------
+# Authors: R. W. Ford and A. R. Porter, STFC Daresbury Lab
+
+'''PSyAD transformations package module'''
+
+from psyclone.psyad.transformations.tangentlinear_error import \
+    TangentLinearError
+from psyclone.psyad.transformations.adjoint_trans import AdjointTransformation
+from psyclone.psyad.transformations.assignment_trans import AssignmentTrans
```

Comment:
I'd like to discuss this before making the change, or not. The reason I did not add Adjoint to the name is because we are in the psyad hierarchy so its kind of implied. It seems a bit of a waste of time to add it. Having said that I can see that it might be confusing if we start using transformations for PSyclone and psyad together if their names are not distinct. On the other hand we could simply do something like `import psyclone.psyad.transformations as adjoint_trans` and then use adjoint_trans.AssignmentTrans so it would then be clear where it is from. So, as you can see I'm in two minds. Is there a killer reason to do what you suggest that I'm missing? Perhaps, psyad will have multiple transformations on an assignment node and we will need to distinguish?

Reply:
I think that your argument that this transformation is within `psyad.transformations` is a good one and am happy to leave it as is.